2_task(magazine)/js/dropdown.js

```diff
@@ -0,0 +1,25 @@
+function myFunction(elem) {
```

Comment:
not good idea to call functions like myFunction
is it possible to use event listeners instead of pushing func to global scope? for each dropdown you need to add onclick in view, it can be avoided